src/route-sync/config/config.go

```diff
@@ -1,47 +1,148 @@
 package config
 
 import (
-	"encoding/json"
+	"fmt"
+	"io/ioutil"
 
+	yaml "gopkg.in/yaml.v2"
+
+	"code.cloudfoundry.org/multierror"
 	cfConfig "code.cloudfoundry.org/route-registrar/config"
-	"errors"
-	"github.com/kelseyhightower/envconfig"
 	uaaconfig "code.cloudfoundry.org/uaa-go-client/config"
 )
 
-// Config contains settings for the route-sync with envconfig annotations
+// ConfigSchema defines the file schema for the YAML configuration of route-sync
+type ConfigSchema struct {
+	NatsServers               []MessageBusServerSchema `yaml:"nats_servers"`
```

Comment:
The `cfConfig.MessageBusServer` is able to map directly without the extra YAML tags